libs/npress/FrontModule/presenters/PagesPresenter.php

```diff
@@ -34,14 +34,21 @@ public function actionDefault($id_page)
       );
     }
 
+    $history = \PagesModel::getThisVersion($id_page,$id_version);
+    $this->template->history = $history;
```

Comment:
Myslím že tu chybí to zobrazení alertu, když je zrovna vidět historická verze.

Jo a `$history` je takové mnohoznačné. Šlo by třeba `$historicVersion`?